Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,218 @@
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - StunBaton
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - Flash
+    - FlashPayload
+    - ForensicPad
+    - ForensicScanner
+    - ClothingGlassesEyesSecurity
+    - FlashlightSeclite
+
+- type: technology
+  name: technologies-ballistic-ammunition
+  id: BallisticAmmunitionTechnology
+  description: technologies-ballistic-ammunition-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    state: base-6
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - MagazineBoxPistol
+    - MagazineBoxPistolRubber
+    - MagazineBoxMagnum
+    - MagazineBoxMagnumRubber
+    - MagazineBoxRifle
+    - MagazineBoxRifleRubber
+    - MagazineBoxLightRifle
+    - MagazineBoxLightRifleRubber
+
+- type: technology
+  name: technologies-ballistic-magazines
+  id: BallisticMagazinesTechnology
+  description: technologies-ballistic-magazines-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/Magazine/Pistol/pistol_mag.rsi
+    state: red-icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - BallisticAmmunitionTechnology
+  unlockedRecipes:
+    - SpeedLoaderMagnum
+    - SpeedLoaderMagnumHighVelocity
+    - SpeedLoaderMagnumRubber
+    - MagazineRifle
+    - MagazineRifleHighVelocity
+    - MagazineRifleRubber
+    - MagazinePistolSubMachineGunTopMounted
+    - MagazinePistol
+    - MagazinePistolHighVelocity
+    - MagazinePistolRubber
+
+- type: technology
+  name: technologies-ballistic-weaponry
+  id: BallisticWeaponryTechnology
+  description: technologies-ballistic-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/SMGs/wt550.rsi
+    state: icon
+  requiredPoints: 20000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+    - BallisticMagazinesTechnology
+  unlockedRecipes:
+    - WeaponRifleLecter
+    - WeaponSubMachineGunWt550
+    - WeaponShotgunEnforcer
+    - WeaponRevolverInspector
+    - WeaponPistolMk58
+
+- type: technology
+  name: technologies-non-lethal
+  id: NonLethalTechnology
+  description: technologies-non-lethal-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/disabler.rsi
+    state: base
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - Disabler
+    - Flashbang
+    - Bola
+    - FoodBoxDonut
```

Comment:
remove

Reply:
remove the recipes or the nonlethal technology altogether?

Reply:
remove the donut box